### PR TITLE
chore(heroku): change moment version to 2.14.1

### PR DESCRIPTION
**Pull request by vonvick**
#### What does this PR do?
This PR deploys the application on heroku
#### Description of Task to be completed?
Successfully deploy application on heroku and create pipeline
#### How should this be manually tested?
The application can be tested by visiting the app https://anbu-rc-staging.herokuapp.com
#### Any background context you want to provide?
Not Applicable
#### What are the relevant pivotal tracker stories?
#137090889 Deploy application on heroku
#### Screenshots (if appropriate)
Not Applicable
#### Questions:
No question

### CODE REVIEW

File: imports/plugins/core/shipping/client/templates/checkout/shipping.js [hunk 1]
```diff
@@ -1,2 +1,88 @@
-Template.flatRateCheckoutShipping.inheritsHelpersFrom("coreCheckoutShipping");
-Template.flatRateCheckoutShipping.inheritsEventsFrom("coreCheckoutShipping");
+// Template.flatRateCheckoutShipping.inheritsHelpersFrom("coreCheckoutShipping");
+// Template.flatRateCheckoutShipping.inheritsEventsFrom("coreCheckoutShipping");
+
+import _ from "lodash";
+import { Cart, Shipping } from "/lib/collections";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Logger } from "/client/api";
```
[the remark this answers is not in the document]
Comment (vonvick) [reply imports/plugins/core/shipping/client/templates/checkout/shipping.js]: fixed

File: imports/plugins/core/shipping/client/templates/checkout/shipping.js [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+});
+Template.flatRateCheckoutShipping.helpers({
+  // retrieves current rates and updates shipping rates
+  // in the users cart collection (historical, and prevents repeated rate lookup)
+  shipmentQuotes: function () {
+    const cart = Cart.findOne();
+    return cartShippingMethods(cart);
+  },
+  // helper to make sure there are some shipping providers
+  shippingConfigured: function () {
```
[the remark this answers is not in the document]
Comment (vonvick) [reply imports/plugins/core/shipping/client/templates/checkout/shipping.js:48]: fixed

File: imports/plugins/core/shipping/client/templates/checkout/shipping.js [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+    const self = this;
+    const cart = Cart.findOne();
+    try {
+      Meteor.call("cart/setShipmentMethod", cart._id, self.method);
+    } catch (error) {
+      throw new Meteor.Error(error,
+        "Cannot change methods while processing.");
+    }
+  }
+});
```
[the remark this answers is not in the document]
Comment (vonvick) [reply imports/plugins/core/shipping/client/templates/checkout/shipping.js]: fixed

**Review by ghost — APPROVED** [review]
approved for merge

**Comment by vonvick** [comment]
remove import statement and add shipping method directly to  imports/plugins/core/shipping/client/templates/checkout/shipping.js so as to enable shipping method on heroku.

**Review by victor-enogwe — APPROVED** [review]
approved

**Pull request merged**